src/components/Carousel/Carousel.tsx

```diff
@@ -48,6 +48,10 @@ export interface CarouselProps {
   indicatorsActiveColor?: string
   indicatorsInactiveColor?: string
   isIndicatorsShadow?: boolean
+  selectable?: boolean
+  pauseOnHover?: boolean
+  onClickItem?: (index: number) => void
```

Comment:
I initially suggested adding the `onClickItem` function, but later realized it is unnecessary. Users can directly set up the `onClick` event handler on each carousel item when passing the items as props

Reply:
totally, changed